templates/parts/event-form.php

```diff
@@ -71,6 +71,17 @@
 				?>
 			</select>
 		</div>
+		<div>
+			<label for="event-attendance-mode"><?php esc_html_e( 'Attendance Mode', 'gp-translation-events' ); ?></label>
+				<input type="radio" id="event-attendance-mode-hybrid" name="event_attendance_mode" <?php echo $event->is_hybrid() ? esc_attr( 'checked' ) : ''; ?> value="hybrid" required>
+				<span title="Attendees can attend remotely and on-site"><?php esc_html_e( 'Hybrid (Remote and On-site)', 'gp-translation-events' ); ?></span>
```

Comment:
If you use a `label` HTML tag like this, the text is clickable to activate the radio box. Please also do this for the other radios.
```suggestion
				<label title="Attendees can attend remotely and on-site"><input type="radio" id="event-attendance-mode-hybrid" name="event_attendance_mode" <?php echo $event->is_hybrid() ? esc_attr( 'checked' ) : ''; ?> value="hybrid" required>
				<?php esc_html_e( 'Hybrid (Remote and On-site)', 'gp-translation-events' ); ?></label>
```